packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.stories.js

```diff
@@ -42,13 +42,17 @@ storiesOf("Atoms|CircleIcon", module)
       },
       iconColor: {
         default: color("iconColor", "#FFFFFF", "Props")
+      },
+      ariaLabel: {
+        default: text("ariaLabel", "Home Icon", "Props")
```

Comment:
```suggestion
        default: text("ariaLabel", "Go to home page", "Props")
```
check example https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute you should use "action" not describe icon inside 👍